modules/stanford_profile_styles/lib/scss/component/node/stanford_opportunity.scss

```diff
@@ -204,8 +207,14 @@
   }
 
   .su-opp-units {
+    padding-bottom: 0;
+
     > div:first-of-type {
       font-weight: $su-font-semi-bold;
+
+      &::after {
+        content: ':';
```

Comment:
why not do this in the template? https://github.com/SU-SWS/stanford_profile/pull/883/files#diff-bda16813a2c4ae5b7f3cd06733f207e444405244cff464bf4906e8f4598a5388R18

Reply:
When I open a fresh GitPod from that branch, my types are in `<div>` and not the list in the template. I don't mind updating to your suggested way.